From: uazo <uazo@users.noreply.github.com>
Date: Fri, 20 Oct 2023 14:53:41 +0000
Subject: Disable prefers-reduced-transparency media query

This feature can be used for fingerprinting as it exposes a user preference
---
 .../blink/renderer/platform/runtime_enabled_features.json5    | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/third_party/blink/renderer/platform/runtime_enabled_features.json5 b/third_party/blink/renderer/platform/runtime_enabled_features.json5
--- a/third_party/blink/renderer/platform/runtime_enabled_features.json5
+++ b/third_party/blink/renderer/platform/runtime_enabled_features.json5
@@ -3003,8 +3003,8 @@
       base_feature: "none",
     },
     {
-      name: "PrefersReducedTransparency",
-      status: "stable",
+      name: "PrefersReducedTransparency",  // disabled by default
+      status: "experimental",              // see https://bugs.chromium.org/p/chromium/issues/detail?id=1424879
     },
     // This feature is deprecated and we are evangelizing affected sites.
     // See https://crbug.com/346236 for current status.
--
2.25.1
